spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/retry/SpringAiRetryAutoConfiguration.java

```diff
@@ -43,7 +43,7 @@
  * @author Christian Tzolov
  */
 @AutoConfiguration
-@ConditionalOnClass(RetryTemplate.class)
+@ConditionalOnClass(TransientAiException.class)
```

Comment:
The condition for retryTemplate should be maintained. Therefore, I suggest the following

```suggestion
@ConditionalOnClass({ RetryTemplate.class, TransientAiException.class })
```